### DISCUSSION TITLE
Colors are different over SSH

### DISCUSSION THREAD

**Discussion by CobaltCause** [helix-editor/helix#6963] [Q&A] [answered]
### Summary

Without SSH:

![image](https://user-images.githubusercontent.com/7003738/236113048-e6111d80-9576-407d-804d-6c7c99653a28.png)

With SSH:

![image](https://user-images.githubusercontent.com/7003738/236112974-df13d888-f7ad-4782-be9f-90cd0151d956.png)

### Reproduction Steps

1. Open a file
2. Notice that it works fine
3. `ssh localhost`
4. Open the same file
5. Notice that it does not work fine

### Helix log

Log is empty.

### Platform

NixOS 23.05.20230430.da45bf6 (Stoat)

### Terminal Emulator

Alacritty 0.12.0

### Helix Version

commit d1a4bd876b3ae646693c0905d7f29b636e2e5033

**Comment by pascalkuthe** [comment]
Over ssh true color support often isn't detected. You can use `:set true-color true` to fix that

**Reply by dist1ll** [reply, marked as answer]
Or in the config file:

```toml
[editor]
true-color = true
```

**Reply by tqwewe** [reply]
I also get different colors when using tmux, despite me having true-color enabled.

![20230504_233436.jpg](https://user-images.githubusercontent.com/16362377/236231687-b24fa43b-af96-4c59-8b22-571f772dff97.jpg)

Both these are editing the same file using the same theme, but the left side is with tmux.

**Reply by CobaltCause** [reply]
Running `:set true-color true` doesn't seem to have any effect for me.

**Reply by tqwewe** [reply]
I solved it with <https://stackoverflow.com/questions/41783367/tmux-tmux-true-color-is-not-working-properly/41786092#41786092>

**Reply by CobaltCause** [reply]
Unfortunately this doesn't apply to me since tmux is not involved in my situation

**Reply by tqwewe** [reply]
Did you try setting true-color to `true` in the config rather than executing it in an already running helix?

**Reply by CobaltCause** [reply]
Oh my bad, I assumed those would have the exact same effect. Setting it in the config file as mentioned above does fix it even though `:set true-color true` doesn't.

**Reply by lheckemann** [reply]
Would it be possible to have setting this at runtime work? `:set` does suggest it in the completions, and apparently there aren't any fundamental issues with changing the setting at runtime since `:config-reload` does work.
